src/state/sync/UserAssetsSync.tsx

```diff
@@ -9,30 +9,25 @@ import { ChainId } from '@/chains/types';
 export const UserAssetsSync = function UserAssetsSync() {
   const { accountAddress, nativeCurrency: currentCurrency } = useAccountSettings();
 
-  const isSwapsOpen = useSwapsStore(state => state.isSwapsOpen);
-
   useUserAssets(
     {
       address: accountAddress,
       currency: currentCurrency,
     },
     {
-      enabled: !isSwapsOpen,
       select: data =>
         selectorFilterByUserChains({
           data,
           selector: selectUserAssetsList,
         }),
       onSuccess: data => {
-        if (!isSwapsOpen) {
```

Comment:
this allows us to update the store in the scenario when a user has opened the swap flow very quickly before it could get synced - we can be on the lookout for any rendering degradations

Reply:
if we are worried about rendering degradations, a happy medium could be to enable this query if either:
1. swaps is not open
2. swaps is open but user assets store is empty